gradoop-flink/src/main/java/org/gradoop/flink/io/impl/csv/functions/CSVLineToElement.java

```diff
@@ -86,20 +89,23 @@ protected Properties parseProperties(String type, String label, String propertyV
 
   /**
    * Parses the CSV string that contains EPGMGraphHead ids.
+   * The csv string is formatted as an array: {@code "[id0,id1,id3,...]"}
    *
    * @param gradoopIdsString The csv token string.
    * @return gradoop ids contained in the string
    */
   protected GradoopIdSet parseGradoopIds(String gradoopIdsString) {
-    String[] gradoopIds = gradoopIdsString
-      .substring(1, gradoopIdsString.length() - 1)
-      .split(CSVConstants.LIST_DELIMITER);
+    // Check for empty id array string and return empty list
+    // Splitting the empty string would result in 1 empty string instead
+    // If string is not empty, parse to GradoopIds
+    List<GradoopId> gradoopIds = gradoopIdsString.equals("[]") ? Collections.emptyList() : Arrays
```

Comment:
Add the empty graph head string `"[]"` to GradoopConstants.

Reply:
How about so?